indigo/distribution.yaml

```diff
@@ -466,6 +466,16 @@ repositories:
       url: https://github.com/GT-RAIL/async_web_server_cpp.git
       version: develop
     status: maintained
+  aubo_robot:
+    doc:
+      type: git
+      url: https://github.com/auboliuxin/aubo_robot.git
+      version: master
+    source:
+      type: git
+      url: https://github.com/auboliuxin/aubo_robot.git
+      version: indigo
```

Comment:
There is no indigo branch on this repository. Looking at the repository there's only the `master` branch so this should be `master`